### PR TITLE
KAN - 36 Refactor Levels Tree

### PR DESCRIPTION
**Pull request by ErickGS28**
### Feature / Bug Description
Update the dropdown to a dynamic tree in the levels section
### Solution
- Use the library "react-d3-tree". 
### Notes
- I had to modify the Update Level Form to load the level information using the new logic of the tree.
- I created a new component "LevelDetails" to show the level information.

### Tickets
- [TICKET](https://one-sm.atlassian.net/browse/KAN-36)
- [TICKET](https://one-sm.atlassian.net/browse/KAN-42)

### Screenshots / Screencasts
image
https://www.youtube.com/watch?v=f4__KEUwwBs

### CODE REVIEW

File: src/pages/level/components/LevelDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        setIsLoading(false);
+      }
+    };
+
+    fetchLevelData();
+  }, [levelId, getLevel]);
+
+  const getStatusText = (status: string): string => {
+    switch (status) {
+      case "A":
```
Comment (imadiaz) [inline src/pages/level/components/LevelDetails.tsx:36]: can we use constants for this?

**Review by imadiaz — COMMENTED** [review]
Its okey, please update the. code according to the Whatsapp changes that we made yesterday, also instead of show the action menu when you click in a node, what do you think if you create like a drawer to handle the actions?, let me show you lie this component  https://ant.design/components/drawer i think that is more cleaner and easy to use

**Comment by imadiaz** [comment]
> I have updated the code using the drawer from ant design, is it ok? image

Sure it makes more sense but lets add more design. i mean we can replace the buttons with the form to have a better design there i think that the colors and orders is not the best one

**Comment by ErickGS28** [comment]
Ok, what colors should I use? The same as the original dropdown? And what color for he details button?. Is it ok if the order of the buttons is : Create Level > Update Level > Detals > Close ?

**Comment by imadiaz** [comment]
> Ok, what colors should I use? The same as the original dropdown? And what color for he details button?. Is it ok if the order of the buttons is : Create Level > Update Level > Detals > Close ?

Let's keep this design. i'll create a new ticket with the updates!

**Review by imadiaz — COMMENTED** [review]
just check if is responsive too

**Review by Isaacjbar — APPROVED** [review]

**Pull request merged**